src/conversions/anyhow.rs

```diff
@@ -111,13 +111,21 @@ use crate::exceptions::PyRuntimeError;
 use crate::PyErr;
 
 impl From<anyhow::Error> for PyErr {
-    fn from(err: anyhow::Error) -> Self {
-        PyRuntimeError::new_err(format!("{:?}", err))
+    fn from(mut error: anyhow::Error) -> Self {
+        // Error chains & error with contexts are skipped
```

Comment:
This comment is slightly confusing to me as a reader; why are they skipped?

I think the following would be clearer:

```suggestion
        // Errors containing a PyErr without chain or context are returned as the underlying error 
```